src/main/java/com/epam/reportportal/jbehave/JBehaveUtils.java

```diff
@@ -78,8 +85,19 @@ public Launch get() {
 			rq.setName(parameters.getLaunchName());
 			rq.setStartTime(startTime);
 			rq.setMode(parameters.getLaunchRunningMode());
-			rq.setTags(parameters.getTags());
+			rq.setAttributes(parameters.getAttributes());
 			rq.setDescription(parameters.getDescription());
+			rq.setRerun(parameters.isRerun());
+			if (!isNullOrEmpty(parameters.getRerunOf())) {
+				rq.setRerunOf(parameters.getRerunOf());
+			}
+			if (null != parameters.getSkippedAnIssue()) {
+				ItemAttributesRQ skippedIssueAttribute = new ItemAttributesRQ();
+				skippedIssueAttribute.setKey(SKIPPED_ISSUE_KEY);
+				skippedIssueAttribute.setValue(parameters.getSkippedAnIssue().toString());
```

Comment:
NPE?

Reply:
where?

Reply:
doesn't getSkippedAnIssue return `Boolean`? 
Just to make sure - is it initialized with some default value?